libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Generators/TemplateEngineLanguageGenerator.cs

```diff
@@ -63,9 +47,9 @@ public TemplateEngineLanguageGenerator(string lgText, string id, Dictionary<stri
         public TemplateEngineLanguageGenerator(string filePath, Dictionary<string, IList<Resource>> resourceMapping)
         {
             filePath = PathUtils.NormalizePath(filePath);
-            this.Id = Path.GetFileName(filePath);
+            this.Id = filePath;
```

Comment:
>this.Id = filePath; [](start = 12, length = 19)

no, I think I would keep Id as the resource.id
Add a new property for the fullname.

Reply:
updated